Dockerfile

```diff
@@ -0,0 +1,31 @@
+# With this Dockerfile, you can create a container image:
+#     $ docker build -f Dockerfile -t bpftool .
+# And then use it:
+#     $ docker run --rm -ti --privileged --pid=host bpftool bpftool prog
+
+FROM ubuntu:22.04 as builder
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN \
+	apt-get update && \
+	apt-get -y install \
+		build-essential \
+		libelf-dev \
+		libz-dev \
+		libbfd-dev \
+		libcap-dev \
+		clang llvm lld \
+		binutils-dev \
+		libiberty-dev \
+		pkg-config \
+		libc6-dev
+
+COPY . /src
+RUN cd /src/src ; \
+	export EXTRA_LIBS="-lbfd -liberty -lz -ldl" ; \
```

Comment:
Could we drop libbfd and dependencies in favour of llvm? Please have a look at https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?h=eb9d1acf634baf6401dfb4f67dc895290713a357 (to be pulled to this mirror at next sync).

Reply:
That's cool! I will try that after the next sync.

Reply:
Just synced with #49.